GNUmakefile.in

```diff
@@ -55,8 +55,19 @@ distclean: realclean
 
 install: install_lib install_headers install_fortran_modules install_pkg_config
 
-uninstall:
-	@echo Uninstalling...
-	$(SILENT) $(RM) -r $(AMREX_INSTALL_DIR)
+uninstall: uninstall_lib uninstall_headers uninstall_fortran_modules uninstall_pkg_config
+	@echo "Uninstalling tmp_build_dir files"
+	$(SILENT) $(RM) $(f77EXETempDir)/*.f $(f77EXETempDir)/*.f90 $(f77EXETempDir)/*.F90 $(objEXETempDir)/*.d $(depEXETempDir)/f90.depends
+	$(SILENT) $(RM) $(srcTempDir)/* $(objEXETempDir)/*.o $(objEXETempDir)/*.mod
+	$(SILENT) $(RM) $(tmpEXETempDir)/*
+	@if [ -d $(TmpBuildDir) ]; then \
+	    $(SILENT) rmdir -p --ignore-fail-on-non-empty $(f77EXETempDir); \
+	    $(SILENT) rmdir -p --ignore-fail-on-non-empty $(objEXETempDir); \
+	    $(SILENT) rmdir -p --ignore-fail-on-non-empty $(depEXETempDir); \
+	    $(SILENT) rmdir -p --ignore-fail-on-non-empty $(srcTempDir); \
+	    if [ -d $(TmpBuildDir) ]; then \
+	         $(SILENT) rmdir -p --ignore-fail-on-non-empty $(TmpBuildDir); \
+            fi; \
+	fi;
```

Comment:
Could this be the following?
```suggestion
uninstall: uninstall_lib uninstall_headers uninstall_fortran_modules uninstall_pkg_config distclean
```

Reply:
I was copying the directory-specific lines from `cleanconfig`, the main difference is the last 3 lines:
https://github.com/AMReX-Codes/amrex/blob/84b99bd8c612c45c4e36b1394265a430799c85d0/Tools/GNUMake/Make.rules#L200-L207

It looks like distclean otherwise just removes the `GNUMakefile` created by configure, which probably ought to be removed for `make uninstall` as well.